### PR TITLE
RPN-1175: add descriptions

### PR DESCRIPTION
**Pull request by mazinden**

### CODE REVIEW

File: api/protocol-api-erc20/openapi.yaml [hunk 1]
```diff
@@ -8,18 +8,26 @@ paths:
...
+          description: "Address of the ERC20 token owner"
           in: path
           required: true
           schema:
             $ref: '#/components/schemas/Address'
+      x-codeSamples:
+        - lang: 'cURL'
+          source: |
+            curl --request GET \
+            --url 'https://ethereum-api-dev.rarible.org/v0.1/erc20/balances/0x60f80121c31a0d46b5279700f9df786054aa5ee5/0x60f80121c31a0d46b5279700f9df786054aa5ee5'
```
Comment (serejke) [inline api/protocol-api-erc20/openapi.yaml:30]: Better off use https://ethereum-api.rarible.org/ ? not dev

File: api/protocol-api-nft-order/openapi.yaml [hunk 2]
```diff
@@ -12,8 +12,10 @@ paths:
       tags:
         - nft-order-ownership-controller
       operationId: getNftOrderOwnershipById
+      description: Returns order ownership by ownership address
       parameters:
         - name: ownershipId
+          description: "Address of the NFT ownership"
```
Comment (serejke) [inline api/protocol-api-nft-order/openapi.yaml:18]: Ownerships don't have any "address"

ownershipId = "$token:$tokenId:$owner",

for example 0xf6793da657495ffeff9ee6350824910abc21356c:102351784535720381921822338631028197690494847592883259167987535571146174431591:0xe2490e9f5657cfefb89c12798167ecd194c8e956

better to say something like "ID of the ownership in format: '$contractAddress:$tokenId:$ownerAddress'

File: api/protocol-model-nft/openapi.yaml [hunk 3]
```diff
@@ -478,12 +478,15 @@ components:
           "$ref": "#/components/schemas/BigInteger"
         uri:
           type: string
+          description: "Uniform Resource Identifier of the token"
         creators:
           type: array
+          description: "Address of the NFT item creators"
```
Comment (serejke) [inline api/protocol-model-nft/openapi.yaml:484]: Address**es**

File: api/protocol-model-nft/openapi.yaml [hunk 4]
```diff
@@ -518,6 +522,7 @@ components:
       properties:
         "@type":
           type: string
+          description: "NFT type for lazy minting"
```
Comment (serejke) [inline api/protocol-model-nft/openapi.yaml:525]: This is a discriminator (internal) field, I suppose it is not necessary to document

File: api/protocol-api-nft-order/openapi.yaml [hunk 5]
```diff
@@ -290,30 +318,36 @@ paths:
...
             type: integer
             format: int32
         - name: showDeleted
+          description: "Include deleted items"
           in: query
           required: false
           schema:
             type: boolean
         - name: lastUpdatedFrom
+          description: "From what time the items were updated. Integer in unix format"
```
Comment (serejke) [inline api/protocol-api-nft-order/openapi.yaml:343]: Is there a better wording for this?
Maybe "Filter condition to return only items that were updated after this date (unix format)" ?

File: api/protocol-api-order/openapi.yaml [hunk 6]
```diff
@@ -1253,27 +1301,31 @@ paths:
       tags:
         - order-aggregation-controller
       operationId: aggregateNftSellByMaker
-      description: Aggregate nft sell order by maker
+      description: Aggregate NFT sell order by maker
       parameters:
         - name: startDate
+          description: "Start data update time"
```
Comment (serejke) [inline api/protocol-api-order/openapi.yaml:1307]: Is there a better wording?
Maybe "Lower time border of data (unix format)" ?

and "Upper time border of data" for the endDate ?

File: api/protocol-api-nft-order/openapi.yaml (hunk 5 above)
Comment (mazinden) [reply api/protocol-api-nft-order/openapi.yaml:343]: yes, that's much better

File: api/protocol-api-order/openapi.yaml (hunk 6 above)
Comment (mazinden) [reply api/protocol-api-order/openapi.yaml:1307]: yes, fixed

**Pull request merged**